Lekce_02_Ukol04/Lucistnik.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Lekce_02_Ukol04
+{
+    internal class Lucistnik
+    {
+        private int PocetSipu;
```

Comment:
Tim, ze je to privatni, tak by to nemelo zacinat velkym pismem. Konvence je ze to zacina `_`
```suggestion
        private int _pocetSipu;
```

Reply:
opraveno